dbclient/WorkspaceClient.py

```diff
@@ -245,6 +247,8 @@ def download_notebooks(self, ws_log_file='user_workspace.log', ws_dir='artifacts
         checkpoint_notebook_set = self._checkpoint_service.get_checkpoint_key_set(
             wmconstants.WM_EXPORT, wmconstants.WORKSPACE_NOTEBOOK_OBJECT)
         ws_log = self.get_export_dir() + ws_log_file
+        notebook_error_logger = logging_utils.get_error_logger(
+            wmconstants.WM_EXPORT, wmconstants.WORKSPACE_NOTEBOOK_OBJECT, self.get_export_dir())
```

Comment:
I think it's better to organize all error logs into one dir?

Reply:
+1. It will be easier for the invokers to find.